website/source/docs/platform/k8s/helm.html.md

```diff
@@ -85,6 +85,14 @@ and consider if they're appropriate for your deployment.
 
   * <a name="v-global-datacenter" href="#v-global-datacenter">`datacenter`</a> (`string: "dc1"`) - The name of the datacenter that the agent cluster should register as. This may not be changed once the cluster is bootstrapped and running, since Consul doesn't yet support an automatic way to change this value.
 
+ * <a name="v-global-pod-security-policies" href="#v-pod-security-policies">`enablePodSecurityPolicies`</a> (`boolean: false`) -
+  This flag controls whether [PodSecurityPolicies](https://kubernetes.io/docs/concepts/policy/pod-security-policy/) are created
```

Comment:
```suggestion
  This flag controls whether Kubernetes creates [PodSecurityPolicies](https://kubernetes.io/docs/concepts/policy/pod-security-policy/)
```

Reply:
Technically this is incorrect- it's the Helm chart (and this option in particular that is being described) that will control whether these get created.

Reply:
Edited, is this more correct? I think that some of these options are configuring Kubernetes and some of them are configuring Consul, so I think its extra worth it to specify which are which.